cmd/updater/subcommands/bootstrap/command.go

```diff
@@ -68,9 +72,16 @@ func boostrapFxWrapper(ctx context.Context, params *cliParams) error {
 }
 
 func bootstrap(ctx context.Context, params *cliParams) error {
-	err := updater.Bootstrap(ctx, params.pkg)
-	if err != nil {
-		return fmt.Errorf("could not install package: %w", err)
+	if params.version == "" {
+		err := updater.Bootstrap(ctx, params.pkg)
+		if err != nil {
+			return fmt.Errorf("could not install package: %w", err)
+		}
+	} else {
+		err := updater.BootstrapVersion(ctx, params.pkg, params.version)
```

Comment:
FWIW this command should never be called while an updater is running. You probably want a command that calls the localapiclient instead for tracer purposes to mimic what will be done by the backend.